src/usr/bin/google_set_multiqueue

```diff
@@ -108,6 +110,16 @@ function is_a3_platform() {
   || "$machine_type" == *"a3-megagpu-8g"* \
   || "$machine_type" == *"a3-edgegpu-8g"* \
   || "$machine_type" == *"a3-ultragpu-"* ]] || return 1
+  return 0
+}
+
+
+# returns 0 (success) if the supplied nic is a Gvnic device.
+function is_gvnic() {
+  local -r nic_name="$1"
+  driver_type=$(ethtool -i $nic_name | grep driver)
+
+  [[ "$driver_type" == *"gve"* ]] || return 1
```

Comment:
Some versions of the driver present as `gvnic` not `gve`

Reply:
Done.